src/setup_node_env/harden.js

```diff
@@ -22,3 +22,7 @@ var hook = require('require-in-the-middle');
 hook(['child_process'], function(exports, name) {
   return require(`./patches/${name}`)(exports); // eslint-disable-line import/no-dynamic-require
 });
+
+hook(['lodash'], function(exports, name) {
```

Comment:
You don't need to set up a new hook, instead you just add `lodash` to the array up above

Reply:
Awesome, thanks Thomas!